.cargo/config.toml

```diff
@@ -4,3 +4,9 @@ OUT_DIR = { value = "target", relative = true }
 [profile.typegraph_core.release]
 opt-level = "z"
 lto = true
+
+# [target.'cfg(all())']
+# rustflags = [
+#   "--cfg",
+#   "tokio_unstable",
```

Comment:
tokio unstable may be required?

Reply:
Another change I shouldn't have committed. The deno repo had it in their `config` because they use [`tokio-metrics`](https://github.com/tokio-rs/tokio-metrics).